playbooks/common/openshift-cluster/upgrades/disable_excluder.yml

```diff
@@ -5,12 +5,13 @@
   tasks:
   - include: pre/validate_excluder.yml
     vars:
-      #repoquery_cmd: repoquery_cmd
-      #openshift_upgrade_target: openshift_upgrade_target
-      excluder: "{{ item }}"
-    with_items:
-    - "{{ openshift.common.service_type }}-docker-excluder"
-    - "{{ openshift.common.service_type }}-excluder"
+      excluder: "{{ openshift.common.service_type }}-docker-excluder"
+    when: enable_docker_excluder | default(enable_excluders) | default(True) | bool
```

Comment:
This duplicates the information about how the enable_docker_excluder and enable_openshift_excluder are set. However, we can fix it later.